hyperparameter_tuning/blazingtext_text_classification_dbpedia/hpo_blazingtext_text_classification_dbpedia.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "Convair was an american aircraft manufacturing company which later expanded into rockets and spacecraft.",
Line #3.        "Berwick secondary college is situated in the outer melbourne metropolitan suburb of berwick .",
</pre><p>Does it matter if period is space-separated or not? <code>spacecraft.</code>  (no space) versus <code>berwick .</code>  (space)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>